docs/introduction/blockchain-representation.rst

```diff
@@ -2,51 +2,51 @@
-So, the output of the Sidechain SDK, when customized by a developer, is a “Node” that implements core functionalities, and the added logic.
+Being a distributed architecture, the sidechain software is delivered as an application that will be compiled/installed on potentially many different independent, networked computers. In blockchain jargon, these computers are called “nodes,” and the term “node” is also generally used to name the blockchain software itself.
+So, the output of the Sidechain SDK, when customized by a developer, is a “node” that implements core functionalities and the added sidechain logic.
 
-A Node consists of 4 main elements: “**History**,” “**State**,” “**Wallet**,” and “**Memory pool**.” Before we get to know these 4 elements we need to know what a “box” is.
+A node consists of 4 main elements: “**History**,” “**State**,” “**Wallet**,” and “**Memory pool**.” Before we get to know these 4 elements we need to know what a “box” is.
 
 Concept of a BOX
 ****************
 
 A box generalizes the concept of Bitcoin’s UTXOs.
-A box is a cryptographic object that can be created with some secret keys. This box can be open \ (spent) by the owner of those secret keys. Once opened by the owner of the secret keys the box may not be opened again.
+A box is a cryptographic object that can be created with some (one private key or multiple private keys?) secret keys . This box can be opened (spent) by the owner(owners ?) of those secret keys. Once opened by the owner (owners ?) of the secret keys the box may not be opened again.
 
 Node Main elements & intro to a "NodeView"
 ******************************************
 
   * **History**
-    * “History” is a blockchain ledger, that is typically a list of Sidechain blocks that were received by the Node, and that have been verified against Consensus rules, and accepted.
+    * “History” is a blockchain ledger, typically a list of sidechain blocks that were received by the node, have been verified against Consensus rules, and accepted.
     
   * **State**
     * “State” is a snapshot of all boxes that haven’t been opened yet. It represents the state at the current chain tip.
     
   * **Wallet**
     * The “Wallet” has two main functionalities:
-      1. It holds the Secret keys that belong to that specific Node.
+      1. It holds the secret keys that belong to that specific node.
       2. It keeps track of objects that are of interest to this specific node, e.g. received coins (output boxes whose secret keys are known by the node) and views of them (e.g. balances).      
   * **Memory Pool**
-    * The “Memory pool” is a list of transactions that are known to the node but have not made it to a Sidechain block yet.
+    * The “Memory pool” is a list of transactions that is known to the node but have not made it to a Sidechain block yet.
```

Comment:
@zennermac 

The “memory pool” is a list of transactions that are known to the node but have not made it to a sidechain block yet.